plugins/database/src/low_level_odbc.cpp

```diff
@@ -129,8 +129,15 @@ logPsgError( HENV henv, HDBC hdbc, HSTMT hstmt, int dbType ) {
                 errorVal = CATALOG_ALREADY_HAS_ITEM_BY_THAT_NAME;
             }
         }
-
-        log_sql::info("{}: SQLState: {}; SQLCODE: {}; SQL Error message: {}", __func__, sqlstate, sqlcode, psgErrorMsg);
+        // Some versions of fmt don't know how to deal with SQLCHAR[].
```

Comment:
can we mark what version of fmt is sufficient... so that once we clear that version everywhere, we can remove this workaround?

Reply:
It's a problem with newer versions, not older versions.

Reply:
oh, well that's upside down :)

let's make that clear in the comment - thanks.